templates/cv-template.aaryan-harvard.html

```
@@ -0,0 +1,565 @@
+  Projects -> Leadership Experience -> Technical Skills. Garamond throughout,
+  thin underline below section headers, narrow margins (0.3in T/B + 0.4in L/R),
+  bullets with action-verb formulas and quantified results, ATS-clean
+  single-column layout.
+
+  Custom user-layer extension to career-ops v1.22.0:
+    - LEADERSHIP substitution token (added in build-cv-html.mjs)
+    - stripEmptySections pattern for LEADERSHIP and CERTIFICATIONS
+      (added in cv-sections-core.mjs)
+    - Career-ops runs in detached HEAD, so the upstream code is not affected.
+-->
+<html class="resume">
```

HTMLHint (1.9.2)

[warning] An lang attribute must be present on <html> elements.

(html-lang-require)